golang/cosmos/app/app.go

```diff
@@ -849,6 +849,8 @@ func unreleasedUpgradeHandler(app *GaiaApp, targetUpgrade string) func(sdk.Conte
 			vm.CoreProposalStepForModules("@agoric/builders/scripts/smart-wallet/build-wallet-factory2-upgrade.js"),
 			// Then, upgrade Zoe and ZCF
 			vm.CoreProposalStepForModules("@agoric/builders/scripts/vats/replace-zoe.js"),
+			// Then, upgrade the provisioning vat
+			vm.CoreProposalStepForModules("@agoric/builders/scripts/vats/replace-provisioning.js"),
```

Comment:
this is more than a test... ah... the commit message says `feat`. The PR title threw me off a bit.